source/test/sgp_mode_test/SGPSymbiont.test.cc

```diff
@@ -56,39 +50,33 @@ TEST_CASE("SGPSymbiont Reproduce", "[sgp]")
       }
     }
 
-    THEN("The symbiont child tracks how its tasks compare to its parent's partner's tasks")
-    {
+    THEN("The symbiont child tracks how its tasks compare to its parent's partner's tasks") {
       REQUIRE(sym_baby->GetCPU().state.task_toward_partner[0] == 0);
       REQUIRE(sym_baby->GetCPU().state.task_from_partner[0] == 1);
 
-      for (unsigned int i = 1; i < CPU_BITSET_LENGTH; i++)
-      {
+      for (unsigned int i = 1; i < CPU_BITSET_LENGTH; i++) {
         REQUIRE(sym_baby->GetCPU().state.task_toward_partner[i] == 0);
         REQUIRE(sym_baby->GetCPU().state.task_from_partner[i] == 0);
       }
     }
     sym_baby.Delete();
   }
 
-  SECTION("Symbiont comparison operators")
-  {
+  TEST_CASE("Symbiont comparison operators", "[sgp]") {

```

Comment:
It looks like this is still nested in the Reproduce test case, I can't see where the { on line 6 closes.